castervoice/lib/const.py

```diff
@@ -11,16 +14,21 @@ class CCRType(object):
     # Rules which were split out of _caster.py:
     "CasterRule", "HardwareRule", "MouseAlternativesRule", "WindowManagementRule",
     # Alternate mouse grid controls:
-    "LegionGridRule", "DouglasGridRule", "RainbowGridRule", "SudokuGridRule",
+    "DouglasGridRule", "RainbowGridRule", "SudokuGridRule",
     # HMC GUI control rules:
     "HMCRule", "HMCConfirmRule", "HMCDirectoryRule",
     "HMCHistoryRule", "HMCLaunchRule", "HMCSettingsRule",
     # GUI Rules
     "HistoryRule", "ChainAlias", "Alias",
     # other common rules
-    "DragonRule", "BringRule", "Again",
+    "BringRule", "Again"
     ]
 
+if sys.platform == "win32":
```

Comment:
In this case DNS grammars would be enabled with Kaldi running on Windows OS. An engine rather than a platform check would be a more robust implementation. 

This can be achieved like the following:

```
engine = get_engine()
if engine._name == 'kaldi':
# or
if engine._name == 'natlink': # DNS

```

Reply:
Fixed.